testv2/e2e/scenario_install.go

```diff
@@ -65,56 +66,66 @@ func (scenario *scenarioInstall) install(t *testing.T) {
 		t.Fatalf("only 1 version is expected to be set, got %v", scenario.versions)
 	}
 
-	clusterName := clusterName()
-
-	if err := scenario.infra.terraform.init(clusterName); err != nil {
+	if err := scenario.infra.terraform.Init(); err != nil {
 		t.Fatalf("terraform init failed: %v", err)
 	}
 
-	if err := retryFn(scenario.infra.terraform.apply); err != nil {
-		t.Fatalf("terraform apply failed: %v", err)
-	}
-
 	t.Cleanup(func() {
 		if err := retryFn(func() error {
-			return scenario.infra.terraform.destroy()
+			return scenario.infra.terraform.Destroy()
 		}); err != nil {
 			t.Fatalf("terraform destroy failed: %v", err)
 		}
 	})
 
-	k1 := newKubeoneBin(
-		scenario.infra.terraform.path,
-		renderManifest(t,
-			scenario.manifestTemplatePath,
-			manifestData{
-				VERSION: scenario.versions[0],
-			},
-		),
-		withKubeoneBin(scenario.KubeonePath()),
-	)
-
-	if err := k1.Apply(); err != nil {
-		t.Fatalf("kubeone apply failed: %v", err)
+	if err := retryFn(scenario.infra.terraform.Apply); err != nil {
```

Comment:
I think in some cases we have to run destroy after failed apply (e.g. on AWS if there are no available subnets).

Reply:
There is a cleanup function registered right before this line

```
	t.Cleanup(func() {
		if err := retryFn(func() error {
			return scenario.infra.terraform.Destroy()
		}); err != nil {
			t.Fatalf("terraform destroy failed: %v", err)
		}
	})
```

Reply:
I'm worried this is going to be annoying. Subnet conflict errors can be quite common on a busy day, which are very reliably resolved by running `destroy` followed by an `apply`. Since we don't do that any longer, we will have to do `/retest` quite a few times. Ideally, it would be nice to prevent those errors from happening, but I'm not sure if that's possible at all.

Reply:
smaller subnets (`/27`) is a way to minimize the impact.